src/components/Camera.vue

```diff
@@ -10,110 +10,110 @@
 </template>
 
 <script>
-export default {
-    name: 'TakePhoto',
-    data(){
-        return {
-        }
-    },
-    mounted(){
-        this.takePhoto();
-    },
-    methods: {
-        //emit the file to the main component
-        fileUpload(name, file){
-            this.$emit('fileupload', name, file);
+    export default {
+        name: 'TakePhoto',
+        data(){
+            return {
+            }
+        },
+        mounted(){
+            this.takePhoto();
         },
+        methods: {
+            //emit the file to the main component
+            fileUpload(name, file){
+                this.$emit('fileupload', name, file);
+            },
 
-        //use tracking js to display the camera and the post it in live
-        takePhoto() {
-            this.displayVideo = true;
-            let video = document.getElementById('video');
-            let canvas = document.getElementById('canvas');
-            let context = canvas.getContext('2d');
-            let tracker = new tracking.ColorTracker(['magenta', 'cyan', 'yellow']);
-            tracking.track('#video', tracker, { camera: true });
-            tracker.on('track', (event) => {
-                context.clearRect(0, 0, canvas.width, canvas.height);
-                event.data.forEach(function(rect) {
-                    if (rect.color === 'custom') {
-                        rect.color = tracker.customColor;
-                    }
-                context.strokeStyle = rect.color;
-                context.strokeRect(rect.x, rect.y, rect.width, rect.height);
-                context.fillStyle = "#fff";
+            //use tracking js to display the camera and the post it in live
+            takePhoto() {
+                this.displayVideo = true;
+                let video = document.getElementById('video');
+                let canvas = document.getElementById('canvas');
+                let context = canvas.getContext('2d');
+                let tracker = new tracking.ColorTracker(['magenta', 'cyan', 'yellow']);
+                tracking.track('#video', tracker, { camera: true });
+                tracker.on('track', (event) => {
+                    context.clearRect(0, 0, canvas.width, canvas.height);
+                    event.data.forEach(function(rect) {
+                        if (rect.color === 'custom') {
+                            rect.color = tracker.customColor;
+                        }
+                    context.strokeStyle = rect.color;
+                    context.strokeRect(rect.x, rect.y, rect.width, rect.height);
+                    context.fillStyle = "#fff";
+                    });
```

Comment:
Indentation problem. This shouldn't be a problem anymore, please install eslint + prettier.